contrib/ros/envs/sawyer_env.py

```diff
@@ -0,0 +1,346 @@
+                                                            self.obj_range,
+                                                            size=2)
+            self._gazebosrv.set_model_pose(
+                model_name='block',
+                new_pose=Pose(
+                    position=Point(
+                        x=object_xypos[0],
+                        y=object_xypos[1],
+                        z=self._initial_model_pos['object0'][2])),
+                reference_frame='world')
+
+    @property
+    def action_space(self):
+        return self._robot.action_space
+
+    @property
+    def observation_space(self):
+        """
+        Returns a Space object
+        """
+        return Box(-np.inf, np.inf, shape=self._get_obs()['observation'].shape)
+
+    # ================================================
+    # Functions that gazebo env asks to implement
+    # ================================================
+    def initial_setup(self, initial_model_pos):
+        # Extract information for sampling goals
+        gripper_pose = self._robot.gripper_pose
+        self.initial_gripper_pos = np.array(gripper_pose['position'])
+        self.initial_gripper_ori = np.array(gripper_pose['orientation'])
+        if self.has_object:
+            self.height_offset = initial_model_pos['object0'][2]
+
+        self._robot.move_to_start_position()
+
+        if self._simulated:
+            # Generate the world
+            # Load the model
+            self._gazebosrv.load_gazebo_sdf_model(
+                model_name='cafe_table',
+                model_pose=Pose(
+                    position=Point(initial_model_pos['table0'][
+                        0], initial_model_pos['table0'][1], initial_model_pos[
+                            'table0'][2])),
+                model_path=osp.join(
+                    rospkg.RosPack().get_path('sawyer_sim_examples'), 'models',
+                    'cafe_table/model.sdf'))
+            self._gazebosrv.load_gazebo_urdf_model(
+                model_name='block',
+                model_pose=Pose(
+                    position=Point(initial_model_pos['object0'][
+                        0], initial_model_pos['object0'][1], initial_model_pos[
+                            'object0'][2])),
+                model_path=osp.join(
+                    rospkg.RosPack().get_path('sawyer_sim_examples'), 'models',
+                    'block/model.urdf'))
+            self._gazebosrv.load_gazebo_sdf_model(
+                model_name='target',
+                model_pose=Pose(
+                    position=Point(gripper_pose['position'][0], gripper_pose[
+                        'position'][1], gripper_pose['position'][2])),
+                model_path=osp.join(
+                    rospkg.RosPack().get_path('sawyer_sim_examples'), 'models',
+                    'target/model.sdf'))
+        else:
+            # TODO(gh/4: Finish 'initial_setup' function for real robot)
+            pass
+
+    # yapf: disable
```

Comment:
Rather than using `# yapf: disable`, just define temporary variables for your 3 random variables.